### PR TITLE
Doc'd use of asgiref.sync adapters with sensitive variables.

### PR DESCRIPTION
**Pull request by carltongibson**

### CODE REVIEW

**Review by felixxm — COMMENTED** [review]
@carltongibson Thanks :+1: I left small suggestions.

File: docs/topics/async.txt [hunk 1]
```diff
@@ -338,3 +338,17 @@ trigger the thread safety checks:
 Rather, you should encapsulate all database access within a helper function
 that can be called with ``sync_to_async()`` without relying on the connection
 object in the calling code.
+
+Use with exception reporting filters
+------------------------------------
```
[the remark this answers is not in the document]
Comment (carltongibson) [reply docs/topics/async.txt:343]: Wasn't sure. My thought was _Otherwise it's ≈ under `sync_to_async()`_ — but not attached to that. 

?

**Comment by carltongibson** [comment]
> Should we mention the second decorator? django.views.decorators.debug.sensitive_post_parameters

I don't think this is necessary, since this is pulled off of the `request` object (rather than from the traceback frame via 🪄)

File: docs/howto/error-reporting.txt [hunk 2]
```diff
@@ -194,6 +194,17 @@ filtered out of error reports in a production environment (that is, where
             def process_info(user):
                 ...
 
+    .. warning:: Use with ``asgiref.sync`` adapters
```
Comment (felixxm) [inline docs/howto/error-reporting.txt:197]: `warning` won't render nice with a title. We can use `admonition` or remove the title.

Comment (carltongibson) [reply docs/howto/error-reporting.txt]: OK, dropped the title. Thanks 👍

**Review by felixxm — APPROVED** [review]
@carltongibson Thanks :+1:

**Pull request merged**